libr/core/rvc.c

```diff
@@ -355,46 +355,65 @@ static bool rm_empty_dir(const char *rp) {
 	return true;
 }
 
-static RList *repo_files(const char *rp) {
-	RList *files = r_file_lsrf (rp);
-	if (!files) {
-		return NULL;
+//This is why anonymous functions are cool
```

Comment:
This is not anonymous. Anonymous means it have no name. And C doesnt support such things. Thats just a static/local function

Reply:
yes lol, that't the point I wish C had Anonymous functions becaus if it did I would have been able to embed dr into repo_files...

Reply:
Remove the comment and find a better name for the function

Reply:
What should that name be?